apps/req.c

```diff
@@ -1177,15 +1182,16 @@ static int add_DN_object(X509_NAME *n, char *text, const char *def,
         }
     }
 
-    if (buf[0] == '\0')
+    if (buf[0] == '\0') {
```

Comment:
Since you're here, remove the extra parens in the return statements.  Just for this block of code, or any other "nearby" code you touch.